dotcom-rendering/cdk/bin/cdk.ts

```diff
@@ -33,14 +33,52 @@ new RenderingCDKStack(cdkApp, 'ArticleRendering-CODE', {
 	guApp: 'article-rendering',
 	stage: 'CODE',
 	domainName: 'article-rendering.code.dev-guardianapis.com',
-	scaling: { minimumInstances: 1, maximumInstances: 2 },
+	scaling: {
+		minimumInstances: 1,
+		maximumInstances: 2,
```

Comment:
Maybe increase the maxInstances to 4 to give it some space? We've seen a few failing deployments in CODE.